### ISSUE TITLE
Regex commands

### ISSUE THREAD

**Issue by camasthecat** [sdkcarlos/artyom.js#20]
It'd be cool if we could use Regex's to make our commands more natural / smart.

**Comment by sdkcarlos** [comment]
@camasthecat  How didn't i think about it ?

Should the code only check if the regexp matches with the recognized text isn't ? The `test` function should do the trick:

```javascript
/^([a-z0-9]{5,})$/.test('abc1');// False, then skip this index

/^([a-z0-9]{5,})$/.test('abc123');// True, then trigger command
```
Then we could add regular expressions with smart commands:

```javascript
artyom.addCommands({
    indexes: [/hello/],
    // Enable wildcards and RegExps
    smart: true,
    action: (i) => {
        artyom.say("Hey, how are you today?");
    }
});
```

**Comment by camasthecat** [comment]
That'd be awesome, yeah! Funnily enough I've never used regex's before, but now I'll learn!!

This looks amazing!

**Comment by sdkcarlos** [comment]
Regular expressions in smart commands are available in Artyom V1.0.5.

[Read the docs for more information](http://docs.ourcodeworld.com/projects/artyom-js/documentation/methods/addcommands)
